service/protocol-samples/http-client-to0-sample/src/main/java/org/fido/iot/sample/To0ClientApp.java

```diff
@@ -71,6 +72,66 @@
       + "1ek15IbeCI5z7BHea2GZGgaK63cyD15gNA==\n"
       + "-----END EC PRIVATE KEY-----";
 
+  private static String sampleOwnerKeyPem = "-----BEGIN CERTIFICATE-----\n"
```

Comment:
Are these the same keys that are used in component sample Owner? If not, we can think of aligning this at a later point of time.

Reply:
Yes.